### PR TITLE
[DEL-47] Set .triangle:hover colour transition to only apply when body.interactive is present

### PR DESCRIPTION
**Pull request by haydende**
* Styles for hovering over triangles (i.e. colour changes and cursor appearance) only applies when `<body>` has `interactive` class applied
*  Circumcircles are also only generated when interactive mode is active

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -132,7 +139,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <br>
 
-[1.3.1]: https://github.com/JRSmiffy/delaunay/compare/1.3.1...1.3.2
+[1.3.3]: https://github.com/JRSmiffy/delaunay/compare/1.3.2...1.3.3
+[1.3.2]: https://github.com/JRSmiffy/delaunay/compare/1.3.1...1.3.2
```
Comment (jrsmth) [inline CHANGELOG.md:143]: Cheers 👍

File: app/index.scss [hunk 2]
```diff
@@ -57,12 +57,14 @@ body {
...
-    fill: rgba(80, 250, 123, 0.1);
+body.interactive {
+  .triangle {
+    transition: 300ms;
+
+    &:hover {
+      cursor: pointer;
+      fill: rgba(80, 250, 123, 0.1);
+    }
   }
```
Comment (jrsmth) [inline app/index.scss:68]: Does this work?

```css
.interactive.triangle {
     transition: 300ms;

     &:hover {
       cursor: pointer;
       fill: rgba(80, 250, 123, 0.1);
     }
}
```

Comment (haydende) [reply app/index.scss:68]: Unfortunately not. `.interactive.triangle` applies to elements with both `interactive` and `triangle` classes applied, and classes aren't passed on to children.

Comment (jrsmth) [reply app/index.scss:68]: Okay, no worries.

**Review by jrsmth — APPROVED** [review]

**Pull request merged**